berechenbarkeit-lib/src/vendors/regex.rs

```diff
@@ -129,7 +138,10 @@ impl RegexVendor {
             None => pos_counter.clone()
         };
         *pos_counter = *pos_counter + 1u32;
-        let vat: f64 = self.get_vat_rate_from_class(groups.name("VAT").unwrap().as_str().trim())?;
+        let vat: f64 = match groups.name("VAT") {
+            Some(class) => self.get_vat_rate_from_class(class.as_str().trim())?,
+            None => self.default_vat_class.unwrap(),
```

Comment:
This should return an error instead of panicking to avoid crashes of axum threads